Apparently the version limit is only required for systemd udev, it works
just fine with our version of eudev.

diff --git a/meson.build b/meson.build
index 2605a65..3550496 100644
--- a/meson.build
+++ b/meson.build
@@ -218,7 +218,7 @@ endif
 
 have_libgudev = get_option('udev')
 if have_libgudev
-  libudev_dep = dependency('libudev', version: udev_req)
+  libudev_dep = dependency('libudev')
   gudev_dep = dependency('gudev-1.0', version: gudev_req)
   udev_dep = dependency('udev')
 
